### ISSUE TITLE
Service Fabric SDK with new .Net Core 2.0

### ISSUE THREAD

**Issue by amarwadi** [Azure/service-fabric#24]
Hello, Are there any recommendations/changes to make the Service Fabric projects work with the new .NET Core? My Service Fabric solution was based on .NET 4.5.2 and included Stateless ASP.NET Web API, Reliable Services, Reliable Actors. I foolishly upgraded everything to use .NET 4.6.1 and also upgraded the Web API to work with .NET Core 2.0 and now am unable to deploy this to the cluster or run it.

Any thoughts? Is there an ETA on when the SDK would be updated to make this work out of the box? I've tried creating a brand new solution and that doesn't seem to work either.

Any help would be very much appreciated.

Anup

**Comment by mkosieradzki** [comment]
I have upgraded everything to Kestrel and it seems to works (also everything is updated to .NET Standard 2.0 and ASP.NET Core 2.0 and is compatible with .NET Core 2.0 and .NET 4.6.1).

WebListener is not going working until next SDK release. (due to package renaming)

**Comment by mkosieradzki** [comment]
Yes, For example use dynamic port numbers + some kind of proxy: for example (https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reverseproxy) or some nginx/haproxy or just proxy your request using stateless app and proxy middleware (preview) https://github.com/aspnet/Proxy.
